examples/step-2/doc/results.dox

```diff
@@ -1,14 +1,43 @@
 <h1>Results</h1>
 
-The program has, after having been run, produced two sparsity
-patterns. We can visualize them by opening the <code>.svg</code> files in a web browser.
+The program has, after having been run, produced two files of
+DoF locations and sparsity patterns each (once for the original numbering
+and once after renumbering), along with one mesh file.
 
-The results then look like this (every point denotes an entry which
-might be nonzero; of course the fact whether the entry actually is
+Let us start with the DoF locations. There is no particularly convenient
+program to visualize this kind of information, but we can resort to
+<a href="http://www.gnuplot.info/">GNUPLOT</a> (one of the simpler visualization
+programs; maybe not the easiest to use since it is command line driven, but
+also universally available on all Linux and other Unix-like systems). The
+command that produces the following pictures reads as follows:
+```
+plot [-.2:2.2][-1.2:1.2] "mesh.gnuplot" with lines, "dof_locations_2.gnuplot" using 1:2:3 with labels point offset 1,1
+```
+This may be cryptic, but it is also not particularly important and you shouldn't
+spend too much time understanding what it does. Rather, the important part is
+to look at what we get as output:
+<table style="width:60%" align="center">
+  <tr>
+    <td><img src="https://www.dealii.org/images/steps/developer/step-2.dof-locations-1.svg" alt=""></td>
+    <td><img src="https://www.dealii.org/images/steps/developer/step-2.dof-locations-2.svg" alt=""></td>
```

Comment:
```suggestion
    <td><img src="https://www.dealii.org/images/steps/developer/step-2.dof-locations-1.png" alt=""></td>
    <td><img src="https://www.dealii.org/images/steps/developer/step-2.dof-locations-2.png" alt=""></td>
```
Almost - the files are now correct but the links in the tutorial still point to the (now nonexistent) SVG files.